package.json

```diff
@@ -20,7 +20,6 @@
     "lint:functions": "cd functions && npm install && npm run lint && cd ..",
     "format": "prettier --write --plugin-search-dir=. .",
     "doc": "jsdoc -c jsdoc.conf.json && mkdir -p docs/images && cp images/rally-logo.png docs/images/",
-    "test:unit": "jest ./tests/unit/",
```

Comment:
heh I think this target was aspirational, I think it's fine to remove until we have unit tests for the front-end code.